README.md

```diff
@@ -3997,8 +3965,14 @@ My name is Vlad!
 ## Hello, kottans!
 Hello, kottans!
 
+
+=======
```

Comment:
Looks like a part of a conflict message, should not be left after the conflict resolution.